redact/data_models.py

```diff
@@ -64,6 +64,7 @@ class JobStatus(BaseModel):
     start_timestamp: Optional[float] = None
     end_timestamp: Optional[float] = None
     estimated_time_to_completion: Optional[float] = None
+    warnings: List[str] = Field(default_factory=list)
```

Comment:
I do not really understand this PR. So we add the warnings field but then we do not make use of it somewhere?

Reply:
- We do already make use of it in _redact_. Our testing is based on the client.
- Otherwise yes, it's not much used yet. That's probably going to be a task for Phoenix soon. Like at least printing the warnings when a job is processed. Or even store results under a different file name so that users become aware of the issues.
- (Well, any developer can make use of the warnings, of course.)